controllers/vaultpkisecret_controller.go

```diff
@@ -374,27 +392,58 @@ func (r *VaultPKISecretReconciler) recordEvent(p *secretsv1beta1.VaultPKISecret,
 	r.Recorder.Eventf(p, eventType, reason, msg, i...)
 }
 
-func (r *VaultPKISecretReconciler) updateStatus(ctx context.Context, p *secretsv1beta1.VaultPKISecret) error {
+func (r *VaultPKISecretReconciler) updateStatus(ctx context.Context, o *secretsv1beta1.VaultPKISecret) error {
 	logger := log.FromContext(ctx)
-	metrics.SetResourceStatus("vaultpkisecret", p, p.Status.Valid)
-	if err := r.Status().Update(ctx, p); err != nil {
+	logger.V(consts.LogLevelDebug).Info("Update status called")
```

Comment:
Is there a trace log level available? Seems like this function would be called pretty frequently.

Reply:
I don't think so, but we easily add a new log level.

Reply:
Updated in d7a5ad7c97a73d39d8f253fca0ae8c3f4965d8d0